packages/web-components/src/select/select.spec.md

```diff
@@ -0,0 +1,100 @@
+# Select
+
+## Component Description
+
+The browser-native select control component allows users to choose one option from a dropdown list. It is commonly used in forms for selecting values from a predefined set of options.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/0vOH481cv1VQyfQIX9JALn/Select?node-id=1319-163&t=nn2xgT4ieHjbkqQW-11)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr appearance: "outline" "underline" "filled-darker" "filled-lighter" | "outline"
+- @attr size: "small" "medium" "large" | "medium"
+- @attr disabled: boolean
+- @attr multiple: boolean
+- @attr required: boolean
+- @attr name: string
+- @attr autofocus: boolean
+- @attr autocomplete: "on" "off" | "off"
+- @attr aria-label: string
+- @attr aria-labelledby: string
+- @attr aria-describedby: string
+
+### Outputs
+
+None
+
+### Events
+
+- @input: Fires whenever the value of the `select` has been changed
+- @change: Fires whenever the user modifies the value of `select`
+
+### Slots
+
+- default: child elements such as '<option>' will be placed in the default slot
```

Comment:
Only `option` elements and `optgroup` are supported in Select, so this should probably specify that.